UI-MVC/Controllers/ProjectController.cs

```diff
@@ -0,0 +1,178 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using BL;
+using DAL.Data_Transfer_Objects;
+using Domain.Identity;
+using Domain.Projects;
+using Domain.Users;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Identity;
+using Microsoft.AspNetCore.Mvc;
+using UIMVC.Models;
+
+namespace UIMVC.Controllers
+{
+    public class ProjectController : Controller
+    {
+        private ProjectManager _mgr;
+        private PlatformManager _mgrPlatform;
+        private readonly UserManager<UIMVCUser> _userManager;
+
+        public ProjectController(UserManager<UIMVCUser> userManager)
+        {
+            _mgr = new ProjectManager();
+            _mgrPlatform = new PlatformManager();
+            _userManager = userManager;
+        }
+
+        #region Index
+
+        public ActionResult Index()
```

Comment:
Doet hetzelfde als localhost:5000/Platform/1 (regardless of welke index als er een platform voor bestaat)